engine/src/main/java/org/terasology/logic/console/commands/ClientCommands.java

```diff
@@ -68,6 +68,22 @@ public String setWorldTime(@CommandParam("day") float day) {
         return "World time changed";
     }
 
+    /**
+     * Permanently sets the current world time for the local player in days
+     * @param day Float containing day to be set
+     * @return String message containing message to notify user
+     */
+    @Command(shortDescription = "Permanently sets the current world time in days", requiredPermission = PermissionManager.CHEAT_PERMISSION)
+    public  String setPermanentWorldTime(@CommandParam("day") float day) {
```

Comment:
As the command no longer sets the world time, but rather pauses the sun, the name should be updated to match.